types/multy/package.json

```
@@ -0,0 +1,6 @@
+{
+    "private": true,
+    "dependencies": {
+        "@types/busboy": "^0.3.1"
```

```suggestion
        "@types/busboy": "^0.2"
```
the package is 5 years old, let's try first proper versioning:
https://github.com/EduardoRFS/multy/blob/master/package.json#L28